src/logs/metadata_engine/metadata_engine.py

```diff
@@ -139,14 +142,27 @@ def _apply_rule(rule, record, parsed_record):
     if rule.aws_loggroup_pattern and "log_group" in record:
         extracted_values = parse_aws_loggroup_with_grok_pattern(record["log_group"], rule.aws_loggroup_pattern)
         record.update(extracted_values)
+    if rule.log_content_parse_type == "json":
+        try:
+            record["log_content_parsed"] = json.loads(parsed_record.get("content", {}))
+        except Exception as ex:
+            logging.log_error_with_stacktrace(ex, f"Encountered exception when parsing log content as json, requested by rule for {rule.entity_type_name}")
+    else:
+        record["log_content_parsed"] = parsed_record.get("content", "")
 
     for attribute in rule.attributes:
         try:
             value = jmespath.search(attribute.pattern, record, JMESPATH_OPTIONS)
             if value:
                 parsed_record[attribute.key] = value
-        except Exception:
-            logging.exception(f"Encountered exception when evaluating attribute {attribute} of rule for {rule.entity_type_name}")
+
+                # attributes with priority are available for the calculation of further attributes
+                if attribute.priority is not None:
```

Comment:
So "priority" only means that it goes (as usual) to output but also additionally to input? 
Priority kinda suggests reordering? But we don't have it. So it's more like "allowReferencing" or something? Just thinking, not strong about this in any way

Reply:
OK I just found sorting :) Still not sure priority is correct word, it only touches ordering, does not say about this important thing that is allowing referencing in other places.

Reply:
Nah, let's leave it, I think now it's not that important